developer/js/libexec/create-override-script-regexp.ts

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env npx ts-node
+import {readFileSync} from "fs";
+import * as path from "path";
+
+// Where to find UnicodeData.txt and Blocks.txt
+const UCD_DIR = path.join("..", "..", "resources", "standards-data", "unicode-character-database");
+
+const SPACELESS_SCRIPT_BLOCKS = new Set([
+  "Myanmar",
+  "Lao",
+  "Thai",
+  "Khmer",
+  "Katakana",
+  "Katakana Phonetic Extensions",
+  // Add more scripts here, as necessary!
+]);
+
+let blockIter = blocks();
+let block = nextBlock()
+
+let elligibleCharacters = [];
+
+// @ts-ignore: TypeScript complains that it can't compile for..of over a
+// generator without --downlevelIteration, but it works anyway?
+for (let {codePoint, generalCategory} of unicodeData()) {
+  if (!block.contains(codePoint)) {
+    block = nextBlock();
+  }
+  console.assert(block.contains(codePoint));
+
+  if (SPACELESS_SCRIPT_BLOCKS.has(block.name) && isLetterOrMark(generalCategory)) {
+    elligibleCharacters.push(codePoint)
```

Comment:
```suggestion
    eligibleCharacters.push(codePoint);
```

(I know the semicolon is not technically required but consistency is good and ASI is an abomination 😏)